app/next-client-app/components/core/sidebar.tsx

```diff
@@ -0,0 +1,166 @@
+                    <Image
+                      width={0}
+                      height={0}
+                      src="/carrot-logo.png"
+                      alt="carrot-logo"
+                      className="mx-3"
+                      style={{ width: "100%", height: "auto" }}
+                    />
+                    Carrot
+                  </div>
+                </Link>
+              </SheetHeader>
+              <div className="h-full">
+                <div className="flex mt-7 flex-col w-full gap-1">
+                  {sidebarItems.links.map((link, idx) => (
+                    <Link key={idx} href={link.href}>
+                      <SidebarButton
+                        variant={pathname === link.href ? "secondary" : "ghost"}
+                        icon={link.icon}
+                        className="w-full"
+                      >
+                        {link.label}
+                      </SidebarButton>
+                    </Link>
+                  ))}
+                </div>
+                <div className="absolute w-full bottom-4 px-1 left-0">
+                  <Separator className="absolute -top-3 left-0 w-full" />
+                  <Drawer>
+                    <DrawerTrigger asChild>
+                      <Button
+                        variant="outline"
+                        className="w-full justify-start"
+                      >
+                        <div className="flex justify-between items-center w-full">
+                          <div className="flex gap-2">
+                            <span>{userName}</span>
+                          </div>
+                          <MoreHorizontal size={20} />
+                        </div>
+                      </Button>
+                    </DrawerTrigger>
+                    <DrawerContent className="mb-2 p-2 w-[350px]">
+                      <div className="flex flex-col space-y-2 mt-2">
+                        <Link href="/accounts/password_change/">
+                          <SidebarButton
+                            size="sm"
+                            icon={Settings}
+                            className="w-full"
+                          >
+                            Change Password
+                          </SidebarButton>
+                        </Link>
+                        <Link href={"/accounts/logout/"}>
+                          <SidebarButton
+                            size="sm"
+                            icon={LogOut}
+                            className="w-full"
+                          >
+                            Log Out
+                          </SidebarButton>
+                        </Link>
+                      </div>
+                    </DrawerContent>
+                  </Drawer>
+                </div>
+              </div>
+            </SheetContent>
+          </Sheet>
+        </div>
+        {/* TODO: Need to confirm again this link */}
```

Comment:
Yes good point - this should now just default to `"/"` (currently this redirects to `/scanreports` anyway, for now ....

Reply:
Thanks! I've changed this to "/"